utils/workspace-editor/README.md

```diff
@@ -0,0 +1,137 @@
+# WorkspaceEditor
```

Comment:
Looks like the LICENSE.md is missing from the package.

Reply:
done